iamctl/cmd/interactive/import.go

```diff
@@ -0,0 +1,134 @@
+/**
+* Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+*
+* WSO2 LLC. licenses this file to you under the Apache License,
+* Version 2.0 (the "License"); you may not use this file except
+* in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package interactive
+
+import (
+	"bytes"
+	"crypto/tls"
+	"fmt"
+	"io"
+	"log"
+	"mime"
+	"mime/multipart"
+	"net/http"
+	"net/textproto"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"github.com/wso2-extensions/identity-tools-cli/iamctl/pkg/utils"
+)
+
+var importCmd = &cobra.Command{
+	Use:   "import",
+	Short: "You can import a service provider",
+	Long:  `You can import a service provider`,
+	Run: func(cmd *cobra.Command, args []string) {
+		importFilePath, errEXL := cmd.Flags().GetString("importFilePath")
+
+		if errEXL != nil {
+			log.Fatalln(errEXL)
+		}
+		importApplication(importFilePath)
+	},
+}
+
+func init() {
+
+	createSPCmd.AddCommand(importCmd)
+	importCmd.Flags().StringP("importFilePath", "i", "", "set the export file name")
+}
+
+func importApplication(importFilePath string) bool {
+
+	importedSp := false
```

Comment:
Was this variable used? Only returned in line 133
And the return values is not used in line 48